src/main/java/uk/gov/digital/ho/hocs/search/api/dto/AddressDto.java

```diff
@@ -1,13 +1,12 @@
 package uk.gov.digital.ho.hocs.search.api.dto;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
Probably doesn't matter that much, but it's preferred to have singular imports so we don't bring the whole of lombok in